components/navbar/index.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import { Menu } from 'semantic-ui-react';
+import { useGoogleMap } from '../../lib/use-google-map';
+import { EventSearcher } from './event-searcher';
+
+export const Navbar = ({ user, sticky }) => {
+  const [address, setAddress] = useState('');
+  const { results } = useGoogleMap({
+    googleMapsApiKey: 'AIzaSyDzZZDbA1GNJ54oT_EmG47ldziOXUmWQjw',
```

Comment:
this is getting flagged from a handful of guards (google cloud, gitguardian). I am aware of this and will be removing it before shipping 😛